docker/main/Dockerfile

```diff
@@ -232,6 +223,16 @@ RUN --mount=type=bind,source=./requirements-dev.txt,target=/workspace/frigate/re
 
 CMD ["sleep", "infinity"]
 
+# Dev Container w/ TRT
+FROM devcontainer AS devcontainer-trt
```

Comment:
This should move into `docker/tensorrt` too, no?

Reply:
No, I don't think there is a reason for it to be there since it's not being used for pushing or building it is only for running the dev container and it complicates the number of targets that need to be available for the hcl file

Reply:
`devcontainer-trt` doesn't even build for me because it depends on `trt-deps` and `trt-wheels`, which the main Dockerfile doesn't know about